src/widgets/search-box/search-box.js

```diff
@@ -37,6 +37,10 @@ const KEY_SUPPRESS = 8;
  * @return {Object}
  */
 
+const defaultQueryHook = function (query, setQueryAndSearch) {
+  setQueryAndSearch(query);
+};
+
```

Comment:
I like named functions too but not sure when they are assigned to a variable, does it bring any difference at runtime? It should not anymore.

```js
var superFn = function(){}
var superFn = function superFn(){}
```